book/03-git-branching/sections/remote-branches.asc

```diff
@@ -80,9 +80,9 @@ To https://github.com/schacon/simplegit
 ----
 
 This is a bit of a shortcut.
-Git automatically expands the `serverfix` branchname out to `refs/heads/serverfix:refs/heads/serverfix`, which means, ``Take my serverfix local branch and push it to update the remote's serverfix branch.''
+Git automatically expands the `serverfix` branchname out to `refs/heads/serverfix:refs/heads/serverfix`, which means, ``Take my `serverfix` local branch and push it to update the remote's `serverfix` branch''.
```

Comment:
Apparently there is [some debate](https://style.mla.org/punctuation-and-quotation-marks/) as to whether periods should go inside quotation marks, in this book we've settled on the "inside rule."

Reply:
Ok. Fixed.